packages/shared/lib/core/wallet/utils/getOutputOptions.ts

```diff
@@ -25,6 +26,19 @@ export function getOutputOptions(
         amount = nativeTokenId ? '0' : bigAmount.toString()
     }
 
+    const assets: Assets = {}
+    if (nativeTokenId) {
+        assets.nativeTokens = [
+            {
+                id: nativeTokenId,
+                amount: '0x' + bigAmount.toString(16),
+            },
+        ]
+    }
+    if (nftId) {
```

Comment:
At the moment wallet.rs doesn't allow to set both `nativeTokenId` & `nftId`. If there is an `nftId`, it should take precedence over the `nativeTokenId`.

Reply:
https://github.com/iotaledger/wallet.rs/blob/feat/issuer-and-sender-to-output/src/account/operations/transaction/prepare_output.rs#L37-L39